### PR TITLE
Update native app-control guide

### PR DESCRIPTION
**Pull request by hjhun**
This patch updates guide for access control with privileges.

Signed-off-by: Hwankyu Jhun <h.jhun@samsung.com>

### CODE REVIEW

**Comment by mijin85cho** [comment]
FYI : This update looks related with #203 under surbhig12's review

File: docs/application/native/guides/app-management/app-controls.md [hunk 1]
```diff
@@ -430,25 +430,19 @@ app_control_result(app_control_h request, app_control_h reply, app_control_resul
...
+> In the application manifest file, the valid operation name format is `http://tizen.org/appcontrol/operation/<verb>`. You cannot use the related macro name, `APP_CONTROL_OPERATION_<VERB>`.
 
 The operation, URI, and MIME type information is used when [resolving the application control](#resolution). The operation information is mandatory, while the URI or MIME type information is optional. Any application requesting a launch of your application must either specify your application ID (for an explicit launch) or have the same operation value and applicable URI and MIME type information (for an implicit launch).
+You can define privileges to restrict your application from launching. To launch your application, the caller application must request for launch permission by defining privileges in the 'tizen-manifest.xml' file.
 
-> **Note**  
+> **Note**
 > The URI or MIME type can contain wildcards, such as '\*', to match against given conditions in the app control:
 >
 > - In the MIME type, you can use 2 types of wildcards: `image/*` and `*/*`.
```
Comment (surbhig12) [inline docs/application/native/guides/app-management/app-controls.md]: In the MIME type, you can use 2 types of wildcards:

Please rephrase to:

In the MIME type, you can use two types of wildcards:

Comment (hjhun) [reply docs/application/native/guides/app-management/app-controls.md]: done

**Comment by mijin85cho** [comment]
Same update with #203. As #203 is approved, I consider this PR is also reviewed and merge it.

**Pull request merged**